app/services/courses/copy_to_provider_service.rb

```diff
@@ -5,7 +5,7 @@ def initialize(sites_copy_to_course:, enrichments_copy_to_course:)
       @enrichments_copy_to_course = enrichments_copy_to_course
     end
 
-    def execute(course:, new_provider:, force: false)
+    def execute(course:, new_provider:, force:)
       return nil unless course.rollable? || force
```

Comment:
the `nil` here is redundant.

Reply:
:face_with_head_bandage: 

We can card this and leave it for another time as it solely tech debt

Reply:
https://trello.com/c/SkOktCFG/608-remove-pointless-return-nil